pkgs/tools/filesystems/rmount/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchgit, makeWrapper, lib, pkgs ? import <nixpkgs> {} }:
+with pkgs;
+
+stdenv.mkDerivation rec {
+  name    = "rmount-${version}";
+  version = "1.0.1";
+  rev = "v${version}";
+
+  src = fetchgit {
+    inherit rev;
+    url    = "https://github.com/Luis-Hebendanz/rmount";
+    sha256 = "1wjmfvbsq3126z51f2ivj85cjmkrzdm2acqsiyqs57qga2g6w5p9";
+  };
+
+  buildInputs = [ stdenv makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/share/man/man1
+    cp ${src}/rmount.man $out/share/man/man1/rmount.1
+    cp ${src}/rmount.bash $out/bin/rmount-noenv
+    cp ${src}/config.json $out/share/config.json
+    chmod +x $out/bin/rmount-noenv
+
+    makeWrapper $out/bin/rmount-noenv $out/bin/rmount \
+    --prefix PATH : ${lib.makeBinPath [ nmap jq cifs-utils sshfs ]}
+  '';
+
+  meta = {
+      homepage = "https://github.com/Luis-Hebendanz/rmount";
+      description = "Remote mount utility which parses a json file";
+      license = stdenv.lib.licenses.mit;
+    };
```

Comment:
[You probably want to add yourself as maintainer (and to maintainers.nix)](https://nixos.org/nixpkgs/manual/#chap-meta). (Doubly so since in addition to maintaining it here, you seem to be the author!)

You also will need to provide the `platforms` attribute. 

> Hydra builds packages according to the platform specified. If no platform is specified, the package does not have prebuilt binaries.